support-workers/src/test/scala/com/gu/support/workers/lambdas/CreatePaymentMethodSpec.scala

```diff
@@ -53,15 +53,18 @@ class CreatePaymentMethodSpec extends AsyncLambdaSpec with MockContext {
     val outStream = new ByteArrayOutputStream()
 
     createPaymentMethod
-      .handleRequestFuture(wrapFixture(createStripeSourcePaymentMethodContributionJson()), outStream, context)
+      .handleRequestFuture(wrapFixture(createStripePaymentMethodContributionJson()), outStream, context)
       .map { _ =>
         // Check the output
         val createSalesforceContactState = Encoding.in[CreateSalesforceContactState](outStream.toInputStream)
 
         createSalesforceContactState.isSuccess should be(true)
         createSalesforceContactState.get._1.paymentMethod match {
           case Left(stripe: CreditCardReferenceTransaction) =>
-            stripe.TokenId should be("1234")
+            withClue("stripe: " + stripe) {
+              stripe.SecondTokenId should be("12345")
+              stripe.CreditCardNumber should be("1234")
```

Comment:
we put different information into zuora for the new stripe gateway, so this test checks different things now.